### ISSUE TITLE
NullPointerException - 1.13 Hotfix 2: Rustic Waters

### ISSUE THREAD

**Issue by MBfromOK** [DaedalusGame/EmbersRekindled#223]
Keep getting a crash report suggesting that Embers Rekindled is at blame for a crash...

I have been trying to figure out what did it.

The most recent things I did in game before the crash was to construct the Ember Activator and break two fluid pipes that were showing both steam and rainbow particles going from my aqueous accumulator to a Caminite reservoir (via a Caminite valve). The game crashes within seconds of loading now.

Here is the stack trace:
```
---- Minecraft Crash Report ----
// Surprise! Haha. Well, this is awkward.

Time: 2020-02-01 15:14:51 CST
Description: Unexpected error

java.lang.NullPointerException
    at teamroots.embers.compat.MysticalMechanicsIntegration.addCapabilityMechanicalDescription(MysticalMechanicsIntegration.java:226)
    at teamroots.embers.compat.MysticalMechanicsIntegration.addCapabilityInformation(MysticalMechanicsIntegration.java:219)
    at teamroots.embers.EventManager.addCapabilityInformation(EventManager.java:438)
    at teamroots.embers.EventManager.onGameOverlayRender(EventManager.java:419)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_964_EventManager_onGameOverlayRender_Post.invoke(.dynamic)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
    at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182)
    at net.minecraftforge.client.GuiIngameForge.post(GuiIngameForge.java:911)
    at net.minecraftforge.client.GuiIngameForge.renderHUDText(GuiIngameForge.java:741)
    at net.minecraftforge.client.GuiIngameForge.renderGameOverlay(GuiIngameForge.java:173)
    at net.minecraft.client.renderer.EntityRenderer.updateCameraAndRender(EntityRenderer.java:1098)
    at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1119)
    at net.minecraft.client.Minecraft.run(Minecraft.java:3942)
    at net.minecraft.client.main.Main.main(SourceFile:123)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28) 
```

If there is more information that is needed or useful let me know?

**Comment by MBfromOK** [comment]
This seems to be related to the Tinker's Lens, I had it equipped in my offhand when I broke the two blocks, upon launching the world the game was slow, but would load until I moved (highlighting another block) or the block I was looking at loaded.  When I quickly switched my offhand to another item and threw the Tinker's Lens on the ground the level loaded as expected.

When I put it on my hotbar to test my theory the level started insta-crashing; I was able to load the level by repeatedly pressing a key to un-highlight the lens, then moved it into normal inventory.

I had downloaded Embers 1.13 hotfix to see if the issue still happened in that version, switching back to newest now.
